hooks/hooks.py

```diff
@@ -14,8 +17,20 @@
 @hooks.hook('config-changed')
 def config_changed():
     """
-    Called whenever our service configuration changes.
+    Called whenever the configuration changes.
```

Comment:
Might document what the function does rather than when it gets called.  IE:

"On the execution of juju event 'config-changed' this function determines the appropriate version and architecture of kubernetes. It installs the appropriate binaries if the version has changed and removes the old binaries. etc etc"